packages/cli/src/commands/app/build.ts

```diff
@@ -27,8 +28,24 @@ export default async (cmd: Command) => {
   });
   await buildBundle({
     entry: 'src/index',
+    parallel: parseParallel(cmd.parallel),
     statsJsonEnabled: cmd.stats,
     config: ConfigReader.fromConfigs(appConfigs),
     appConfigs,
   });
 };
+
+function parseParallel(
+  parallel: boolean | string | number | undefined,
+): ParallelBundingOption {
+  if (parallel === undefined || parallel === null) {
+    return undefined;
+  } else if (typeof parallel === 'boolean' || typeof parallel === 'number') {
+    return parallel;
+  } else if (parallel === 'true' || parallel === 'false') {
```

Comment:
This case is already covered by the previous if statement

Reply:
This handles the cases where someone does `--parallel true` or `--parallel false`. Although it would be more appropriate to just omit the boolean and say `--parallel` or `--no-parallel` which would result in the same. Maybe I should just remove support for explicit booleans.

Reply:
Yeah but you just tested `typeof parallel === 'boolean'`; this particular if statement will never ever match because the one above already took it :)

Reply:
This is checking for the strings `'true'` and `'false'`

Reply:
Oh derp :( sorry